app/aws-lsp-json-binary/package.json

```diff
@@ -8,9 +8,9 @@
     },
     "scripts": {
         "compile": "tsc --build",
-        "package-x64": "pkg --targets node18-linux-x64,node18-win-x64,node18-macos-x64 --out-path bin --compress GZip .",
+        "pkg": "pkg --targets node18-linux-x64,node18-win-x64,node18-macos-x64 --out-path bin --compress GZip .",
```

Comment:
so we still want to keep this usage of pkg right?

Reply:
I think if we are only keeping it in for testing, we can remove these and use the bundled js files for testing